kubernetes/calico-node.env

```diff
@@ -0,0 +1,6 @@
+ETCD_AUTHORITY=127.0.0.1:4001
+HOSTNAME=dev
+IP="10.0.2.15"
```

Comment:
Yeah that is a better solution.

This was another area I wanted to clean up since it is likely tied to my local setup.